CONTRIBUTING.md

```diff
@@ -197,6 +197,57 @@ QObject notToMentionThatWeUseCamelCase;
 
 E.g. https://github.com/tux3/qTox/blob/master/src/misc/flowlayout.cpp
 
+## Documenataion
+
+If you added new function, add doxygen comment before implementation.
```

Comment:
`If you added a new function, also add a doxygen comment before the implementation.
If you changed an old function, make sure the doxygen comment is still correct. If it doesn't exist add it.`